tests/src/Interop/MarshalAPI/NativeLibrary/NativeLibrary.cpp

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+#include <stdio.h>
+#include <xplatform.h>
+
+extern "C" DLL_EXPORT int RetValue = 100;
+
+extern "C" DLL_EXPORT int __cdecl NativeSum(int a, int b)
```

Comment:
Why cdecl? Unless there is a compelling reason, we should not need to define the calling convention unless testing something related to it.

Reply:
Feel free to mark it as `STDCALLMETHOD` if you really want to be specific.